### ISSUE TITLE
CREATE VIEW: SELECT: cannot use non GROUP BY column '%1' in query results without an aggregate function

### ISSUE THREAD

**Issue by yzchang** [MonetDB/MonetDB#7023]
**Describe the bug**
CREATE VIEW doesn't seem to recognise that a generated column does have an aggregate function:
```
create table tst (k integer  not null, name char(20)  not null);
-- this works
select min(k) from tst group by name;
-- but putting it in a VIEW doesn't
create view v1 as select max(k) from tst group by name;
select * from v1; -- returns SELECT: cannot use non GROUP BY column '%1' in query results without an aggregate function
```

**Expected behavior**
No error but 0 tuple.

**Software versions**
 - MonetDB version number [v11.39.8 (hg id: d1c774305959)]
 - OS and version: [e.g. Ubuntu 20.04]
 - Mercurial source

**Comment by yzchang** [comment]
test added: sql/test/BugTracker-2020/Tests/view_with_aggr_column.Bug-7023.sql

**Comment by yzchang** [comment]
When this issue is revolved, all commented-out queries in `sql/test/Tests/as3ap` (`mtest` branch) should be enabled.

**Comment by yzchang** [comment]
Hai @PedroTadim Is there a corresponding fix for this?

**Comment by swingbit** [comment]
@yzchang I think it's a duplicate of #7013 ?

**Comment by yzchang** [comment]
It's similar, but not the same since this "select ... from <view>" query still fails with the fix for #7013 (i.e. changeset 97cb2149ff53). The fix for this issue is [108a4f6c62d2](https://dev.monetdb.org/hg/MonetDB?cmd=changeset;node=108a4f6c62d2).
